package.json

```diff
@@ -36,6 +36,7 @@
     "gulp-watch": "^4.3.11",
     "istanbul": "^0.4.5",
     "jsdoc-babel": "^0.2.1",
+    "jsdoc-baseline": "git+https://github.com/sugarcrm/jsdoc-baseline.git",
```

Comment:
This is already published, so we can use:
https://www.npmjs.com/package/@sugarcrm/jsdoc-baseline